App.js

```diff
@@ -18,13 +19,19 @@ export const UnconnectedApp = () => (
   <FlagsProvider>
     <MenuProvider>
       <Theme use='default'>
-        <PermissionsProvider>
+        {isGPS ? (
```

Comment:
I like pulling the conditional up the tree a lot! though what do you think about passing the permission provider the tracing strategy itself? I think that help with future iteration were we introduce additional tracing strategies.

something like:
```
          <PermissionsProvider tracingStrategy={gps}>
            <ExposureHistoryProvider>
              <Entry />
            </ExposureHistoryProvider>
          </PermissionsProvider>
```

though this might be a good refactor for later.

Reply:
I am going to keep this as is for now I think. I want to see how the exposure history provider works out too. But I think it will be an easy refactor in the future.